app/frontend/tests/unit/utils/constants.spec.js

```diff
@@ -100,7 +100,8 @@ describe('Constants', () => {
 
   it('Regex has the right values defined', () => {
     expect(constants.Regex).toEqual({
-      EMAIL: '^[a-zA-Z0-9!#$%&\'*+/=?^_`{|}~-]+(?:\\.[a-zA-Z0-9!#$%&\'*+/=?^_`{|}~-]+)*@(?:[a-zA-Z0-9](?:[a-zA-Z0-9-]*[a-zA-Z0-9])?\\.)+[a-zA-Z0-9](?:[a-zA-Z0-9-]*[a-zA-Z0-9])?$'
+      EMAIL: '^[a-zA-Z0-9!#$%&\'*+/=?^_`{|}~-]+(?:\\.[a-zA-Z0-9!#$%&\'*+/=?^_`{|}~-]+)*@(?:[a-zA-Z0-9](?:[a-zA-Z0-9-]*[a-zA-Z0-9])?\\.)+[a-zA-Z0-9](?:[a-zA-Z0-9-]*[a-zA-Z0-9])?$',
+      GUID: '^[{]?[0-9a-fA-F]{8}-([0-9a-fA-F]{4}-){3}[0-9a-fA-F]{12}[}]?$',
```

Comment:
Should we be introducing the GUID check via a regex? We could be leveraging the existing `uuidValidate` and `uuidVersion` methods from the `uuid` library to achieve the same result.
Ref: https://www.npmjs.com/package/uuid#uuidvalidatestr

Reply:
Used uuid instead